### PR TITLE
Landing page news room

### PR DESCRIPTION
**Pull request by ghost**
### What we did:

- Create article model and controller
- Add title and list of article to index.html.haml
- Configure FactoryBot

### CODE REVIEW

File: features/step_definitions/basic_steps.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Given("I am on the landing page") do
+  visit root_path
+end
+
+Given("we have the following {string}") do |string, table|
```
Comment (magnus-thor) [inline features/step_definitions/basic_steps.rb]: remove non used variable `string`

File: spec/models/article_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  let(:article) {create(:article)}
+
+  describe 'Database table' do
+    it { is_expected.to have_db_column :headline}
+    it { is_expected.to have_db_column :content}
```
Comment (magnus-thor) [inline spec/models/article_spec.rb]: Mind the code formatting. you have a space after the opening curly but not before the enclosing curly.

**Review by magnus-thor — CHANGES_REQUESTED** [review]

**Review by magnus-thor — APPROVED** [review]

**Review by GeorgeJLewis — APPROVED** [review]

**Comment by GeorgeJLewis** [comment]
LGTM

**Review by magnus-thor — APPROVED** [review]
Looks good

**Comment by tochman** [comment]
This one needs to be reviewed by a team member before merge

File: app/models/article.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class Article < ApplicationRecord
+  validates :headline, presence: true
+  validates :content, presence: true
```
Comment (tochman) [inline app/models/article.rb:3]: These validations can be simplified
```
validates_presence_of :headline, :cintent
```

**Pull request merged**